airflow/cli/commands/task_command.py

```diff
@@ -97,7 +97,7 @@ def _get_dag_run(
     dag: DAG,
     create_if_necessary: CreateIfNecessary,
     exec_date_or_run_id: str | None = None,
-    session: Session | None = None,
+    session: Session = NEW_SESSION,
```

Comment:
Not needed here since we don't always use it (for example when `create_if_necessary="memory" and exec_date_or_run_id is None`)

Reply:
fixed in local. i will push soon.